### ISSUE TITLE
Edgenext checkpoints on 224X224 resolution.

### ISSUE THREAD

**Issue by NKSagarReddy** [mmaaz60/EdgeNeXt#17]
Hi,

Could you also share the checkpoints of the Edgenext models(Edgenext-S specifically) trained on 224X224 images(results were mentioned in the paper)?

I would like to compare the metrics with other SOTA models which have been pretrained on 224X224 imagenet-1k resolution images.

**Comment by NKSagarReddy** [comment]
Could you also share how you were able to find different checkpoints like small, x_small, xx_small which perform really well?
Did you use any NAS/Pruning strategy?

**Comment by mmaaz60** [comment]
Hi @NKSagarReddy,

Apologies for the delayed reply. Unfortunately, I couldn't locate the checkpoints at `224x224` resolution. I will share in case if I found it.

For `x_small` and `xx_small` models, we design the architectures to keep the parameters comparable to MobileViT (i.e. 2.3M parameters for x_small & 1.3M for xx_small). Also, when down scaling the architecture from standard EdgeNeXt model to smaller model, we reduce both the depth and width of the network to achieve a good speed-accuracy trade-off. Further, we experimented with different augmentations and find out that the advanced augmentations like `RandAugment` is not required for `small` and `extra small` models.

I hope this will be helpful. Please let me know if you have any questions. Thank you.

**Comment by NKSagarReddy** [comment]
Thank you for the info!

Could you also specify which teacher model you have used for obtaining the USI checkpoints using distillation?

**Comment by NKSagarReddy** [comment]
Thank you for the info!

Could you also share what teacher models were used for distilling EdgeNeXt small and base models for the USI checkpoints?

**Comment by mmaaz60** [comment]
Hi @NKSagarReddy,

We use the same recipe as in [Solving ImageNet](https://arxiv.org/abs/2204.03475) paper for distillation experiments. The teacher network used is [TresNet-L](https://miil-public-eu.oss-eu-central-1.aliyuncs.com/model-zoo/USI/tresnet_l_v2_83_9.pth). I hope this will help. Thanks
